src/meta/src/barrier/mod.rs

```diff
@@ -195,57 +245,125 @@ where
             hummock_manager,
             metrics,
             env,
+            is_recovery: Arc::new(AtomicBool::new(false)),
+            is_build_actor: Arc::new(AtomicBool::new(false)),
+            command_ctx_queue: Arc::new(RwLock::new(VecDeque::new())),
+            in_flight_barrier_nums,
         }
     }
 
-    pub async fn start(barrier_manager: BarrierManagerRef<S>) -> (JoinHandle<()>, Sender<()>) {
-        let (shutdown_tx, shutdown_rx) = tokio::sync::oneshot::channel();
-        let join_handle = tokio::spawn(async move {
-            barrier_manager.run(shutdown_rx).await;
+    /// start two loop (inject and collect)
```

Comment:
May want to add docs to describe the return value.

Reply:
okk